WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -113,28 +119,69 @@ public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bun
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        mUsernamesContainer = (ViewGroup) getView().findViewById(R.id.usernames);
+        mUsernamesContainer = (ViewGroup) view.findViewById(R.id.usernames);
+        mUsernamesContainer.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                EditTextUtils.showSoftInput(mUsernameEditText);
+            }
+        });
 
         String role = mRole;
         if (role == null) {
             role = loadDefaultRole();
         }
 
-        mUsernameEditText = (EditText) getView().findViewById(R.id.invite_usernames);
+        mUsernameEditText = (MultiUsernameEditText) view.findViewById(R.id.invite_usernames);
+        mUsernameEditText.setText(" ");
+        mUsernameEditText.setSelection(1);
+
+        mUsernameEditText.setOnSelectionChangeListener(new MultiUsernameEditText.OnSelectionChangeListener() {
+            @Override
+            public void onSelectionChanged(int selStart, int selEnd) {
+                if (selEnd == 0) {
+                    //whitespace was removed from EditText
+                    if (TextUtils.isEmpty(mUsernameEditText.getText())) {
+                        //removing the last username
+                        List<String> list = new ArrayList<>(mUsernameButtons.keySet());
+                        if (!list.isEmpty()) {
+                            String username = list.get(list.size() - 1);
+                            removeUsername(username);
+                        }
+                        //return the whitespace
+                        mUsernameEditText.setText(" ");
+                    }
+                    mUsernameEditText.setSelection(1);
+                }
+            }
+        });
+
         mUsernameEditText.addTextChangedListener(new TextWatcher() {
+            private boolean shouldIgnoreChanges = false;
+
             @Override
-            public void beforeTextChanged(CharSequence s, int start, int count, int after) {}
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+            }
 
             @Override
             public void onTextChanged(CharSequence s, int start, int before, int count) {
-                if (mUsernameEditText.getText().toString().endsWith(" ")) {
+                if (shouldIgnoreChanges) //used to avoid double call after calling setText from this method
+                    return;
```

Comment:
Let's wrap this in braces to respect the coding style (see: https://source.android.com/source/code-style.html#use-standard-brace-style). Thanks!